website/docs/docs/authentication/legacy.md

```diff
@@ -0,0 +1,490 @@
+---
+title: Authentication
+sidebar_label: Authentication (Cognito)
+---
+
+:::info
+This article applies to accounts that use Cognito for authentication. This includes all accounts before Dec 11th 2023. For customers on SmartBear ID see [this guide](./main).
```

Comment:
It includes all accounts before Dec 11th, 2023. For customers on SmartBear ID see [this guide].